modules/ROOT/pages/policy-mule3-add-remove-headers.adoc

```diff
@@ -6,75 +6,75 @@ endif::[]
 
 API Manager supports several policies for injecting or removing HTTP headers from a request or response. The policies take effect before sending the request or response.
 
-The add request header policy injects one or more headers into a request from the client to the backend service, or overrides the value of an existing header.
-The remove request header policy prevents receipt of one or more specified headers sent from the client to the backend service.
+Add Request Header policy injects one or more headers into a request from the client to the backend service, or overrides the value of an existing header.
```

Comment:
maybe we can merge line 9 with 12. In both lines (9 and 12), the policy does not override the values, it adds new ones, the result will be a request with the old values plus the new ones.
E.g: Request with header "foo=value1" if we add the header "foo=value2" with the policy, the result will be a request with header "foo=value1, value2"